.travis.yml

```diff
@@ -54,7 +54,7 @@ matrix:
         # Sphinx is pinned so that new versions that introduce new warnings won't suddenly cause build failures.
         # (Updating the version is fine as long as no warnings are raised by doing so.)
         # The theme used by the docs is stored separately, so we need to install that as well.
-        - python -m pip install sphinx==2.2.0 blurb python-docs-theme
+        - python -m pip install sphinx==2.2.0 blurb 'git+https://github.com/python/python-docs-theme.git@fc9f203dd79587c8aed2b822cf7401518cd65bb5#egg=python-docs-theme'
```

Comment:
should this now be python-docs-theme==2020.1

Reply:
The usual build pulls the latest version, so it should pull it in and be fine.  This PR was just meant as a quick check before release to make sure the regular build wasn't going to blow up :)